strabo/static/js/map.js

```diff
@@ -24,9 +24,9 @@ $(document).ready(function(){
     var map = make_map('map');
     add_tile_to(map);
 
-    map.addControl( new L.Control.Compass() );/////////////////////////////////
+    flickity_init();
 
-    flickety_init();
+    map.addControl( new L.Control.Compass() );/////////////////////////////////
```

Comment:
the "/////" was me while in development - sorry about that, probably we should add a comment stating that the Leaflet Compass won't work if your device doesn't have a compass